mne/channels/channels.py

```diff
@@ -403,6 +403,8 @@ def pick_channels(self, ch_names, copy=False):
         -----
         .. versionadded:: 0.9.0
         """
+        if not isinstance(ch_names, (tuple, list, np.ndarray)):
+            ch_names = [ch_names]
```

Comment:
I'm fine either way. There are other places that we upconvert a single arg to a list. But if we do it here, we should at least have another check like:
```
if not isinstance(ch_names, (tuple, list, np.ndarray)):
    if not isinstance(ch_names, string_types):
        raise TypeError(...)
    ch_names = [ch_names]
```
But @agramfort if you don't like the upconversion, we don't need to have it.